ludwig/visualize.py

```diff
@@ -731,34 +1420,45 @@ def compare_classifiers_predictions(
 
 
 def compare_classifiers_predictions_distribution(
-        predictions,
-        ground_truth,
-        field,
+        preds_per_model,
+        gt,
```

Comment:
replcase in all function signatures `preds_per_model`, `probs_per_model` and `gt` with `predictions_per _model`, `probabilities_per_model` and `ground_truth`. As those arguments have docstrings and are exposed, I prefer them to be as explicit as possible. Internally in the functions they can be shortened for brevity, but for the exposed ones I prefer them to be longer and clear.